src/__tests__/store/message/reducer.test.ts

```diff
@@ -0,0 +1,74 @@
+import { messageActionTypes, IMessageState } from "../../../src/store/message/types";
+import { messageReducer } from "../../../src/store/message/reducer";
+
+const initialState: IMessageState = {
+    message: undefined,
+    confirmation: undefined
+};
+const testState : IMessageState = {
```

Comment:
The initial state and testing state shouldn't be the same, right? We want to actually test that something is happening rather than just passing undefined around. update the tests to actually test for sample message and confirmation strings please in all tests.